proofs/hol_light/arm/Makefile

```diff
@@ -69,11 +69,25 @@ OBJ = mlkem/mlkem_ntt.o mlkem/mlkem_intt.o
 # According to
 # https://developer.apple.com/documentation/xcode/writing-arm64-code-for-apple-platforms,
 # x18 should not be used for Apple platforms. Check this using grep.
+ifneq ($(OSTYPE_RESULT),Darwin)
 $(OBJ): %.o : %.S
 	$(Q)[ -d $(@D) ] || mkdir -p $(@D)
 	cat $< | $(PREPROCESS) | $(SPLIT) | grep -v -E '^\s+.quad\s+0x[0-9a-f]+$$' | $(ASSEMBLE) -o $@ -
 	$(OBJDUMP) $@ | ( ( ! grep --ignore-case -E 'w18|[^0]x18' ) || ( rm $@ ; exit 1 ) )
 	cat $< | $(PREPROCESS) | $(SPLIT) | $(ASSEMBLE) -o $@ -
+else
+$(OBJ): %.o : %.S
+	@if ! (which gobjcopy >/dev/null); then \
+	  echo "gobjcopy not found. Consider installing it via 'brew install binutils', and put its parent directory (e.g. /opt/homebrew/Cellar/binutils/{VERSION}/bin) in your PATH.";\
```

Comment:
```
/opt/homebrew/opt/binutils/bin
```
is better, since that is created as a symlink to the latest version.

Reply:
Good point! Will add.